addons/web/static/tests/views/view_dialogs/export_data_dialog_tests.js

```diff
@@ -635,7 +635,19 @@ QUnit.module("ViewDialogs", (hooks) => {
                 }
                 if (route === "/web/export/get_fields") {
                     if (!args.parent_field) {
-                        return Promise.resolve(fetchedFields.root);
+                        return Promise.resolve([
```

Comment:
I had to change that test since fields data no longer come from the columns of the list, but from this rpc, so "exportable" was now required in the response